integration-cli/docker_utils.go

```diff
@@ -458,6 +460,32 @@ func (d *Daemon) sock() string {
 	return fmt.Sprintf("unix://%s/docker.sock", d.folder)
 }
 
+func (d *Daemon) getBaseDeviceSize(c *check.C) int64 {
+
+	infoCmdOutput, _, err := runCommandPipelineWithOutput(
+		exec.Command(dockerBinary, "-H", d.sock(), "info"),
+		exec.Command("grep", "Base Device Size"),
+	)
+	c.Assert(err, checker.IsNil)
+	BasesizeSlice := strings.Split(infoCmdOutput, ":")
+	Basesize := strings.Trim(BasesizeSlice[1], " ")
+	Basesize = strings.Trim(Basesize, "\n")[:len(Basesize)-3]
+	BasesizeFloat, err := strconv.ParseFloat(strings.Trim(Basesize, " "), 64)
```

Comment:
There is a difference between parsing and casting. `strconv.ParseInt()` would throw an error if a string with `.` ("12.59") is passed to it, which mean a base device size of `12.59` cannot be directly parsed into an int. We need to parse it as a float and cast it into an `int64`.